content/04.study.md

```diff
@@ -424,6 +424,92 @@ summarized above also apply to interfacial contact prediction for protein
+Additionally, common types of high-throughput screens for PPIs, such as the yeast two-hybrid, can have issues with high rates of false positive results [@doi:10.1186/s12964-015-0116-8 @doi:10.1002/pmic.200800150].
+The imperfections and costs associated with many experimental PPI screening methods have motivated an interest in high-throughput computational prediction.
+
+This section will focus on advances in *de novo* PPI prediction.
+Many machine learning and deep learning approaches to PPI have focused on text mining to extract known relationships from the literature [@doi:10.1016/j.jbi.2007.11.008 @arxiv:1706.01556v2].
+Early *de novo* prediction approaches used a variety of statistical and machine learning tools on structural and sequential data, sometimes with reference to the existing body of protein structure knowledge.
+In the context of PPIs — as in other domains — deep learning shows promise both for exceeding current predictive performance and for circumventing limitations from which other approaches suffer.
+
+One of the key difficulties in applying deep learning techniques to binding prediction is the task of representing peptide and protein sequences in a meaningful way.
+DeepPPI [@doi:10.1021/acs.jcim.7b00028] made PPI predictions from a set of sequence and composition protein descriptors using a two-stage deep neural network.
+Sun et al. [@doi:10.1186/s12859-017-1700-2] applied autocovariances, a coding scheme which returns uniform-size vectors describing the covariance between variables at various positions.
+A stacked autoencoder paired with a softmax classifier then gave predictions as probabilities of interaction.
+Wang et al. [@doi:10.1039/C7MB00188F] used deep learning as an intermediate step in PPI prediction.
+They first extracted sequence descriptors called Zernike moments, which have often been used in image analysis.
+A stacked sparse autoencoder was then used to reduce feature dimensions and noisiness before a novel type of classification vector machine made PPI predictions.
+
+Beyond predicting whether or not two proteins interact, Du et al. [@doi:10.1016/j.ymeth.2016.06.001] showed that a tandem stacked-autoencoder/deep-neural-network method could be used to predict residue contacts for the interfacial regions of interacting proteins.
+A combination of a hidden Markov model with Fisher scores yielded uniform-length features for each residue. Their method significantly exceeded classical machine learning accuracy.
+
+Because many studies used predefined higher-level features, one of the benefits of deep learning — automatic feature extraction — is not fully leveraged.
+More work is needed to determine the best ways to represent raw protein sequence information so that the full benefits of deep learning as an automatic feature extractor can be realized.
+
+#### MHC-peptide binding
```

Comment:
What level header should this be?  Is it part of the PPI section or a new section?

Reply:
I had thought of it as a sub-header under the PPI section, but I think it could go either way. Do you have a preference/recommendation?

Reply:
I also think it could go either way.  If you change it to a `###` header, there aren't any other changes needed.

If you keep it as a `####` sub-header, then I suggest following the `Promoters and enhancers` sub-section as an example.  Specifically, let's add a new general header `### Protein-protein interactions and networks` (or a similar general phrase) and change `Protein-protein interactions` and `PPI networks and graph analysis` to be `####` sub-headers.